<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udpSocket</class>
 <widget class="QDialog" name="udpSocket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>udpSocket</string>
  </property>
  <widget class="QLineEdit" name="m_message">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_start">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开始广播</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_send">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_port"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
